std/node/string_decoder_test.ts

```diff
@@ -66,16 +66,16 @@ Deno.test({
     assertEquals(decoder.end(), "");
 
     decoder = new StringDecoder("base64");
-    assertEquals(decoder.write(Buffer.from("EFBFBDE2", "hex")), "77+94g==");
-    assertEquals(decoder.end(), "4gAA");
+    assertEquals(decoder.write(Buffer.from("EFBFBDE2", "hex")), "77+9");
+    assertEquals(decoder.end(), "4g==");
 
     decoder = new StringDecoder("base64");
-    assertEquals(decoder.write(Buffer.from("F1", "hex")), "8Q==");
+    assertEquals(decoder.write(Buffer.from("F1", "hex")), "");
     assertEquals(decoder.write(Buffer.from("41F2", "hex")), "8UHy");
     assertEquals(decoder.end(), "");
 
     decoder = new StringDecoder("base64");
-    assertEquals(decoder.text(Buffer.from([0x41]), 2), "QQ==");
+    assertEquals(decoder.text(Buffer.from([0x41]), 2), "");
```

Comment:
No opinion on whether it's a good idea to switch implementations but I can confirm that these are indeed the expected outputs.

Reply:
Blame it on me, I wrote the original tests :P